docs/modules/operation/pages/deep-dive/service-assurance/introduction.adoc

```diff
@@ -2,41 +2,41 @@
 [[ga-service-assurance]]
 = Service Assurance
 
-This section will cover the basic functionalities how {page-component-title} tests if a service or device available and measure his latency.
+This section describes how {page-component-title} tests the availability of a service or device and measures its latency.
 
-In {page-component-title} this task is provided by a _Service Monitor_ framework.
-The main component is _Pollerd_ which provides the following functionality:
+In {page-component-title} a service monitor framework fulfils these tasks.
+The main component, Pollerd, provides the following functionality:
 
-* Track the status of a management resource or an application for availability calculations
-* Measure response times for service quality
-* Correlation of node and interface outages based on a <<deep-dive/service-assurance/critical-service.adoc#ga-service-assurance-critical-service, Critical Service>>
+* Tracks the status of a management resource or an application for availability calculations.
+* Measures response times for service quality.
+* Correlates node and interface outages based on a <<deep-dive/service-assurance/critical-service.adoc#ga-service-assurance-critical-service, critical service>>.
 
 The following image shows the model and representation of availability and response time.
 
 .Representation of latency measurement and availability
```

Comment:
Since the previous line describes the image, I don't think we need the alt text.

```suggestion
```